docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +49,169 @@ described below, and relative to the joint TMREL.
+For each of the country-derived relative risk surfaces, the 500 g and 2-week gestational age joint bin
+with the lowest risk was identified. This bin differed within each country dataset. To identify the
+universal 500 g and 2-week gestational age category that would serve as the universal TMREL, all bins that were identified as the TMREL was chosen. This is cat55 (40-42ga, 3500-400g) and cat56 (40-42ga, 4000-4500g)
+
+.. note::
+   the TMREL categories listed in GBD 2017 risk appendix is wrong.  
+
+Causes that are affected by LBWSG
+---------------------------------
+
++----------+---------------------------------------------------------+
+| Cause id | Cause                                                   | 
++==========+=========================================================+
+|  302     | diarrheal diseases                                      |
++----------+---------------------------------------------------------+
+|  322     | lower respiratory tract infections                      |
++----------+---------------------------------------------------------+
+|  328     | upper respiratory tract infections                      | 
++----------+---------------------------------------------------------+
+|  329     | otitis media                                            |
++----------+---------------------------------------------------------+
+|  333     | pneumococcal meningitis                                 |
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | Neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+Restrictions
+++++++++++++
+
```

Comment:
We should note here that this only applies to the early neonatal and late neonatal age groups (Age group IDs 2 and 3)

Reply:
ok will add the ids 2-early and 3- late

Reply:
Cool :)